src/Sentry.AspNetCore/SentryAspNetCoreOptions.cs

```diff
@@ -43,6 +43,20 @@ public bool IncludeRequestPayload
         /// </remarks>
         public RequestSize MaxRequestBodySize { get; set; }
 
+        /// <summary>
+        /// Flush on completed request
+        /// </summary>
+        public bool FlushOnCompletedRequest { get; set; }
+
+        /// <summary>
+        /// How long to wait for the flush to finish, in seconds. Defaults to 2 seconds.
+        /// </summary>
+        public TimeSpan FlushTimeout
+        {
+            get => FlushTimeout;
```

Comment:
This is saying when you call `FlushTimeout`'s getter, it'll call `FlushTimeout`'s getter.

Please just change it to:

`public TimeSpan FlushTimeout { get; set; } = TimeSpan.FromSeconds(2);`

This creates a normal auto-property while defining the default value of it to `TimeSpan.FromSeconds(2);`.
It's equivalent to setting it once via the constructor, like:

```
public TimeSpan FlushTimeout { get; set; }`
public SentryOptions() 
{
    FlushTimeout = TimeSpan.FromSeconds(2);
}
```